ftw/simplelayout/browser/client/test/toolbox.test.js

```diff
@@ -72,6 +72,26 @@ suite("Toolbox", function() {
       });
       assert.deepEqual(addedNodes, [1, 2, 4]);
     });
+
+    test("toolbox components are sorted asc", function() {
+      var toolbox = new Toolbox({
+        layouts: [0],
+        components: {
```

Comment:
The components are defined in alphabetical order... for manful testcase you should define them not alphabetically.